playbooks/remove-drive.yml

```diff
@@ -0,0 +1,56 @@
+- hosts: localhost
+  gather_facts: False
+  any_errors_fatal: True
+  vars_prompt:
+    - name: selected_node
+      private: False
+      prompt: |-
+        Please provide the node name you want drive to be remove
+        {% for node in groups['kube-node']|default([])|sort %}
+          {{ node }}
+        {% endfor %}
+    - name: drive_to_remove
+      private: False
+      prompt: "Please provide a drive name (/dev/xxx or xxx format)"
```

Comment:
I'm a bit confused by this. What does 'xxx format' mean? I guess I'll figure that out from reading the code further down, but may not be clear to someone simply using the tool.

Reply:
Could someone come with a proposal ? I'm failing short finding message for interacting with humans -_-